client/src/components/layout/Navbar.js

```diff
@@ -28,6 +28,12 @@ class Navbar extends React.Component {
 
     const authLinks = (
       <ul className="navbar-nav ml-auto">
+        <li className="nav-item">
+          <Link to="/streams" className="nav-link">
+            Streams
+            <i className="fas fa-sign-out-alt ml-2" />
```

Comment:
Remember To Change Icon From Sign Out To Something That Resembles Streams. We Use FontAwsome For Icons, Check Out Their Website https://fontawesome.com/